doc/source/methods/Similarity.ipynb

```diff
@@ -0,0 +1,269 @@
+{
```

Comment:
<p>I would try to be consistent with the docstrings convention for referring to python modules, variables etc. (pytorch -&gt; <code>pytorch</code>  -- add ticks)</p><p> </p><p>Feels that the definition of grad-dot, grad-cos, grad-asym-dot is already defined above. Maybe we can refer to them? Not sure how is better</p><p> </p><p>Usually the loss -&gt; Usually, the loss</p><p> </p><p>Backend framework. ‘tensorflow’ or ‘pytorch’. -&gt; Backend framework: <code>'tensorflow'</code> | <code>'pytorch'</code> ?</p><p> </p><p>  pytorch device. For example ‘cpu’ or ‘cuda’ -&gt; <code>pytorch</code> device: <code>'cpu'</code> | <code>'cuda'</code>    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/650/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>